source/src/microapp/cs_Microapp.cpp

```diff
@@ -83,6 +83,8 @@ void Microapp::loadApps() {
 	}
 
 	for (uint8_t index = 0; index < g_MICROAPP_COUNT; ++index) {
+		_started[index] = false;
```

Comment:
I would put this in `::init` or even in the constructor to reduce the time that the values are undefined.

Reply:
```	bool _started[g_MICROAPP_COUNT] = {};```